src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Shipped.txt

```diff
@@ -992,8 +992,6 @@ static NuGet.Commands.Restore.Utility.PackageSpecFactory.GetPackageSpec(NuGet.Co
 ~static NuGet.Commands.RestoreSpecException.Create(string message, System.Collections.Generic.IEnumerable<string> files) -> NuGet.Commands.RestoreSpecException
 ~static NuGet.Commands.RestoreSpecException.Create(string message, System.Collections.Generic.IEnumerable<string> files, System.Exception innerException) -> NuGet.Commands.RestoreSpecException
 ~static NuGet.Commands.RestoreSummary.Log(NuGet.Common.ILogger logger, System.Collections.Generic.IReadOnlyList<NuGet.Commands.RestoreSummary> restoreSummaries, bool logErrors = false) -> void
-~static NuGet.Commands.RestoreTargetGraph.Create(NuGet.RuntimeModel.RuntimeGraph runtimeGraph, System.Collections.Generic.IEnumerable<NuGet.DependencyResolver.GraphNode<NuGet.DependencyResolver.RemoteResolveResult>> graphs, NuGet.DependencyResolver.RemoteWalkContext context, NuGet.Common.ILogger log, NuGet.Frameworks.NuGetFramework framework, string runtimeIdentifier) -> NuGet.Commands.RestoreTargetGraph
-~static NuGet.Commands.RestoreTargetGraph.Create(System.Collections.Generic.IEnumerable<NuGet.DependencyResolver.GraphNode<NuGet.DependencyResolver.RemoteResolveResult>> graphs, NuGet.DependencyResolver.RemoteWalkContext context, NuGet.Common.ILogger logger, NuGet.Frameworks.NuGetFramework framework) -> NuGet.Commands.RestoreTargetGraph
```

Comment:
I only see the `RestoreTargetGraph.Create` APIs being used in NuGet.Commands (same assembly) and NuGet.Commands.Test, which I assume has InternalsVisibleTo.

So, if we're going to break these APIs, can we make them interal, to get them out of the public API analyzer?

I don't know why a 3rd party app would have a package reference on NuGet.Commands, but even if they did, I assume that this API wouldn't be useful for them to call.